testdata/src/key_forbidden/key_forbidden.go

```diff
@@ -0,0 +1,30 @@
+package key_forbidden
+
+import "log/slog"
+
+const (
+	snakeKey = "foo_bar"
+)
+
+func tests() {
+	slog.Info("msg")
+	slog.Info("msg", "foo-bar", 1)
+	slog.Info("msg", "foo_bar", 1)           // want `keys include forbidden values`
```

Comment:
Can we tell the user which key is forbidden? It could be confusing if I've added 5 attrs.

I also wonder if we should allow the configuration to say why it's forbidden so the user doesn't just slightly change the key name. That would probably require the configuration to be a map which might be more complicated than it's worth.

Reply:
> Can we tell the user which key is forbidden? It could be confusing if I've added 5 attrs.

Agreed, that would be useful. The report message could be something like `foo_bar is forbidden and should not be used`.

> That would probably require the configuration to be a map which might be more complicated than it's worth.

Let's keep it a simple slice for now, we can extend later. If a team needed to document the reason for forbidding certain keys, comments in the config would probably be enough.

Reply:
Updated in bce4e31c07313a64d973ac244d41fafb985751d2.
I quoted the key name to distinguish empty strings